src/core/lib/iomgr/tcp_windows.h

```diff
@@ -36,12 +36,13 @@
 #ifdef GRPC_WINSOCK_SOCKET
 #include "src/core/lib/iomgr/endpoint.h"
 #include "src/core/lib/iomgr/socket_windows.h"
+#include "src/core/lib/iomgr/tcp_generic_options.h"
 
 /* Create a tcp endpoint given a winsock handle.
  * Takes ownership of the handle.
  */
 grpc_endpoint* grpc_tcp_create(grpc_winsocket* socket,
-                               grpc_channel_args* channel_args,
+                               const grpc_tcp_generic_options& options,
```

Comment:
Might as well just remove this parameter, since it never gets used.

Reply:
Done.